docker/Dockerfile.base

```diff
@@ -1,21 +1,13 @@
 FROM {{ hail_ubuntu_image.image }}
 
-# source: https://cloud.google.com/storage/docs/gsutil_install#linux
-RUN curl --remote-name https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-334.0.0-linux-x86_64.tar.gz && \
-    tar -xf google-cloud-sdk-334.0.0-linux-x86_64.tar.gz && \
-    curl --remote-name https://dl.k8s.io/release/v1.21.14/bin/linux/amd64/kubectl && \
-    install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
-ENV PATH $PATH:/google-cloud-sdk/bin
-
 RUN hail-apt-get-install \
     git \
     htop \
     unzip bzip2 zip tar \
-    wget curl \
```

Comment:
We only use `wget` in `Dockerfile.worker` and we ought to just use curl there. (We also install a bunch of deps in azure that seem unnecessary? cmake? pkg-config? TLS things? shouldn't libfuse-dev be sufficient? We should probably just delete all this and update to blobfuse2).